import/import.py

```diff
@@ -66,6 +67,8 @@ def assert_run_id_format(run_id):
                     help='how to find ip address, <ipify|meta>')
 parser.add_argument('--run-id', help='Distinctive run ID to give to '
                     'this build. Defaults to planet date YYMMDD.')
+parser.add_argument('--skip-snapshot', type=lambda x: bool(strtobool(x)),
```

Comment:
Use the `action='store_true'` shortcut for this. See https://docs.python.org/2/library/argparse.html#action.